src/resolvers/ArticleResolver.ts

```diff
@@ -35,6 +40,14 @@ class ArticleResolver {
     return ArticleRepo.getTrendingArticles(since, limit);
   }
 
+  @FieldResolver((_returns) => Number)
+  async trendiness(@Root() article: Article): Promise<number> {
+    const presentDate = new Date().getTime();
+    // Due to the way Mongo interprets 'article' object,
+    // article['_doc'] must be used to access fields of a article object
+    return article['_doc'].shoutouts / (presentDate - article['_doc'].date.getTime()); // eslint-disable-line
```

Comment:
How are article's dates determined? My concern is that once this app has many users and it just so happens that someone is looking for trendy articles just as an article is released, we get a divide by 0 error. I don't know how precise getTime() is, but maybe just add one in the denominator to be 100% safe.

Reply:
I did consider this possibility. I decided to keep it how it was because the chances a divide by 0 error happen are very low to almost 0. 1. getTime() is very accurate so it would detect even a small difference in time to milliseconds. 2. Inevitably a good amount of time will go by from the date thats assigned to an article on rss, to then parsing it and inputting in db through rss feed, and then querying it for trending articles. HOWEVER, you make a good point, if we can avoid it why not...

Reply:
also what units are left when taking a difference of two date objects like this? Not wholly relevant for the queries because it just needs relative separation between `trendiness` values to sort on, but if we ever want to _display_ this number its worth considering the size of the net number and how we can tune that with the quotient and divisor.

Reply:
Agreed, I'll put some thought into this, current trendiness values are pretty ugly and not conducive for comparison or usage if the need ever comes. Altering the metric to have some sort of readable scale for internal Volume purposes could be rlly useful.